drivers/test-fw/src/bin/hmac_tests.rs

```diff
@@ -1015,9 +1017,9 @@ test_suite! {
     test_hmac4,
     test_hmac_kv_multiblock,
     test_hmac5,
-    test_kdf0,
-    test_kdf1,
-    test_kdf2,
+    test_kdf0_hmac384,
```

Comment:
Do we need to add tests for hmac512?

Reply:
Yes; We'll add that as a separate change though.